### DISCUSSION TITLE
Is open source right for the Metaverse?

### DISCUSSION THREAD

**Discussion by RedEagleP1** [omigroup/omigroup#149] [General]
I run a group called [OCEM](https://discord.gg/2sVsZ6NC6B) and we are exploring open source. 

I have a set of views and theories, including questions about if open source is the ideal method to connect people in the Metaverse, which I would love feedback on. 

Note: The points are written to be a starting point for OCEM internal communication on these issues and are _not_ finalized ideas. They are somewhat controversial, but this is intentional, as we find that taking strong stands helps spark discussion. 
** **

This article is long. Use [TTS reader](https://ttsreader.com/) to have it read to you. 
 
### **# 1. The Metaverse is not possible with mouse and keyboard**

As Metaverse enthusiasts, it's important to remember that we are likely the top 0.1% of computer users. Most people are confused by technology, overwhelmed by options and worried if they push the wrong button they will destroy something. 

Recently, Chrome added the copy and paste function to the menu of Chrome because so many people misunderstood how to actually copy and paste. 

Since the Metaverse simulates the countless possibilities of physical life, the complexity of the user interface gets out of hand quickly.

I remember testing out Second Life and accidentally taking off my pants and being unable to put them back on for 20 minutes.

Clearly, mouse and keyboard would not be a sufficient way of interacting with the metaverse.
****

### **# 2. No significant social organization arises out of putting people in a virtual space**

18 years ago, Second Life came out and the hype around the Metaverse future was just as real back then as it is today. You can visit the empty government buildings, brand showcases and  cities to get a taste of the forgotten glory.

Time and time again people have been put together in a virtual space, and we still went back to physical life for real connections.

We have to ask ourselves, why did we leave virtual worlds and go back to physical life?

OMI Note: _I see Second Life for example as having failed at creating the Metaverse._ 
****

### **# 3. The nature of virtual spaces leads to shallow relationships**

The great benefits of any relationship are usually found in the most challenging operations of cooperation, marriage is a great example. Dating is exciting but sharing a life with someone is deeply sacrificial and yet, the benefits of marriage are enormous. 

In  digital worlds we scarcely invest as heavily in relationships as in physical life. When we can go anywhere, be with anyone it's easy to constantly look for greener grass. 

This problem is accentuated by the ability to change your name/avatar at will. When people can burn others and simply change their identity to escape consequences, they do. Investing is a real reputation and a single identity is the first step to solving this issue. 

** **
### **# 4. The Metaverse will arise from a gaming world**

In digital worlds, we find ourselves mixed in with people from different cultures who have different values and interests. By contrast, in physical life we are naturally segmented with people who are more likely to share our values and ideas. 

Gaming has the power to break ice between people and therefore has intrinsic utility to the Metaverse.

By using gamification we can organize and connect people of similar interests and values in a virtual space. 

Not only that, gaming has the power to get people to relax. Human beings don’t develop meaningful friendships when forced into it. This is what social games and “virtual life” games often get wrong. Actual gameplay builds the context we need to relax and have a conversation to find people we are compatible with. 

** **
### **# 5. Reciprocating facial expressions are necessary for cooperative human behavior**

If you've ever lifted a baby and smiled at it, you know that facial expressions are deeply embedded in how we communicate. When we hurt someone's feelings, their facial expressions changes and, as we reciprocate their facial expressions, our body produces the same emotions as we make that individual feel. This acts as a natural tempering of our behaviour that we don't yeah when we communicate over the internet.

### **# 6. Customization is the enemy of usability**

It's fascinating to bump into so many people who share the idea that no corporation should control the future of human interaction. However, it's laughable that we're using the same ill-advised ideas that allowed Facebook, Google and apps to dominate when it comes to this next chapter. 

One of the reasons that Facebook overtook MySpace was that MySpace allowed people to customize their profile infinitely. This led to a myriad of different user interfaces, confusing navigation and a poor user experience. 

Just imagine a myriad of virtual worlds all with their own user interface, controls and standards. The experience of travelling from world to world would be like being born all over again and having to learn how to read and write. 

Therefore, the virtual world must have a single user experience to reach mass adoption. 

** **
### **# 7. Item interoperability is unlikely**

I keep hearing from people who believe that in the future, NFT items will be taken between games and virtual worlds. I sometimes wonder if any of these people have actually designed a video game. 

I'm in the process of making one now and  the thing you discover very quickly is that everything in a video game is intentional. 

For example; if you have a character in your world, all the items in your world fit the style, height and width of that character. 

Let’s say you introduced a car from one game into another. That car could: 

(1) Not fit the style of the game
(2) Break the game balance, giving the player an advantage
(3) Not fit how light reflects in the game
(4) Have a different control interface 
(5) Be too wide for the road
(6) Have the steering wheel on the wrong side 
(7) Create uncalculated physical results, resulting in a car that sends other cars to space

And the list goes on. 

** **
### **# 8. The network effect would eventually push everyone to a single virtual world**

A long time ago, there was a single company that dominated the telephone market. To call anybody who was the customer of the company you needed to have a phone that was with that same company, phones were not interoperable. Today, if you want to contact somebody on Facebook, you need Facebook to do so. This is how social networks defend their intellectual property.

Meta executive Jason Rubin: 

“The first metaverse that gains real traction is likely to the be the last,” Rubin wrote. “We must act first, and go big, or we risk being one of those wannabes.” - CNBC

The network effect is a scary prospect because it means that there will likely be only one Metaverse to rule them all in the end.

** **
### **# 9. AR and not VR will make the Metaverse mainstream**

Virtual reality headsets have been pitched as a game changer for years now. However, very few people feel comfortable obstructing the entirety of their vision to enter our “geek world''. I think we have to accept that even though we love the medium it's unlikely to have mass adoption anytime soon. 

Augmented reality will allow people to continue their lives as they begin to adopt the new virtual way of doing things. 

CEO of Apple on AR: 

“...one of these very few profound technologies that we will look back on one day and ask, ‘How did we live our lives without it?’”

** **
### **# 10. Distributed governance is necessary to limit censorship**

If any one platform is responsible for the content on the Metaverse, it will be torn apart by that responsibility. Human beings have never agreed on morals and values and to one person black is white and white is black. We are slowly coming out of an incredible era of tolerance which is abnormal for human beings. 

We have spent thousands of years of our history destroying each other over needless inconsequential arguments about small religious differences etc. 

As power moves to the virtual space instead of the physical, everyone is going to be pushing the power broker, the Metaverse platform, to impose their sense of morality on others. 

Eventually these arguments will inevitably tear apart the Metaverse unless responsibility for individual nodes is decentralized. 

** **
### **# 11. The Metaverse has the power to become a dystopian nightmare** 

As the network effect brings everyone to a single Metaverse, the potential for the abuse of power will be astronomical. The Metaverse will understand where you are, who you are with and even what you look at. 

AI will be able to process that data to better understand you than you yourself and data is the first step toward oppression. 

Utilizing all this data, powerful individuals will have the ability to suppress dissent, identify people of contrarian opinion etc. 

Moreover, if we begin to fool ourselves that we can agree on morals and values, those who disagree with the mainstream will live in constant fear of losing their friends, job and opportunities if they are banned from the Metaverse. 
 
 ** **
### **# 12. Decentralization will not aide data protection** 

The web as we know it today is a well distributed system, however, it provides no real protection for our data. Every website constantly scrambles after our data and what companies know about us is incalculable. Fortunately, the amount of data makes driving conclusions difficult right now but in the future AI will solve that problem. 

If decentralization isn't protecting us, there is no reason it will protect us in the Metaverse. 

** **
### **# 13. Censorship is necessary to create community** 

Every space in the physical world has some sort of rules to follow. A library asks us to be quiet, a classroom to be respectful and a workplace to be professional. In the same way creators of spaces must be empowered to moderate the spaces they watch over to create a sense of identity in the space which will help it facilitate connections. 

** **
### **# 14. Open source _may_ not be the answer**

With the advantages of usability of having only one way of doing things in a virtual space, embrace, extend, extinguish takes on a whole new meaning. 

It's too easy for those who would profit from our collaborative efforts to use those efforts to create a singular Metaverse which is more usable and then use the “import but don’t export” model to bring in the best of virtual worlds without allowing anything out of their world. 

Total freedom everywhere will just lead to chaos. 
** **
** **
** **
** **
** **
** **
### Although I would like your feedback on all the points, I would love to point out how the most relevant points pertain to open source:

Please understand that I mention these with the idea that I will be corrected and I hope to be corrected. 

4. The Metaverse will arise from a gaming world
6. Customization is the enemy of usability
7. Item interoperability is unlikely
8. The network effect would eventually push everyone to a single virtual world
12. Decentralization will not aide data protection

### # 4. The Metaverse will arise from a gaming world

Since the metaverse will arise out of a gaming world, we need a publisher of games to make the metaverse. I know that sounds convoluted but creating fun is a very difficult art and the number one comment I get from inhabitants of these metal versus is they don't know what the point of staying. One day it's like the internet and there will be a lot of points to stay there but in the meantime you need to have people cooperating and collaborating and that happens through gaming very often and gaining is a great way to break the ice between people so it's a really necessary step.

This is partially why there's so many metaverses and nothing has really taken off, they are not fun. I know I'm talking to the Second Life crowd mostly, but let's be honest, Second Life was positioned to be an incredible metaverse but it fell way below expectations and I believe the reason for that is behavioral psychology and not technology. 

### # 6. Customization is the enemy of usability

Developers tend to love lots of options, while users hate them. 

Apple is famous for taking away all the possibilities and their consumers are fanboys of products that have been falling behind for a long time (I am an Apple iOS user and I prefer it to Android). 

Gaming consoles are a great example in that it takes away the functionality of your computer that you are buying from them in order to give you much less features and charge you a price for that. There is literally no reason for gaming consoles to exist except to satisfy users that are confused by computers. This does not include Switch. 

Linux is a mine field full of distributions and is a terrible user experience on desktop. No one is developing for a platform with so many distributions because it is simply ludicrous. 

Open source games I have so many issues bringing in users because the people that develop the experience are the hardcore users and those hardcore users cater to hardcore play. I play a game called ZeroK daily because I am a hardcore gamer… it's more of a religion than a game because it's so complicated. 

### # 7. Item interoperability is unlikely

Here is another point where I think that developers vastly underestimate how confused people are by visiting virtual worlds. A virtual world has a different way of using its controls. Every virtual world has different gravity, controls, and standards (like how to know what an NPC is).

When people are divorced from the people who use their product and live in a little bubble where everyone speaks their language and their way of thinking they tend to make horrendous mistakes concerning usability. 

Time and time again open source projects fail at usability and user acquisition due to a low budget and due to what I call the “developer bubble effect.” 

Each one of us lives in a bubble and we all experience the world in a totally unique way. We start out in life thinking that everyone is like us. However, there's a very strong distinction between people who understand things in a logical sequence and people who understand things in a very vague way. 

I'm definitely the latter, and it's very common for social people to be the latter. The reason for this is because human beings are incredibly diverse and people who are great with social skills are great at jumping between two different worlds without context. 

Most great engineers understand the world in a logical sequence of events. They have a different way of visualizing the world (they see things very differently) where they understand all the little in between bits where most of us don't and don't want to. This is why I often programmers are slow to speak compared to others. They like to know what they're saying and they're like to know how to make sure to say it very precisely. This is also why Reddit and stackoverflow are really annoying places to be for most people because everyone is so darn precise. Programmers need to be precise and being precise is too slow when it comes to jumping from person to person quickly and making connections and contacts like people like me do.

Our weekly meetings are a good example. We have tons of people and very few of them speak and that's because developers are by and large less social than your average person. 

### # 12. Decentralization will not aide data protection

I don't want the Metaverse to look anything like the web because the web did nothing for our privacy. The web model, in my mind, will definitely lead to oppression when put in the context of a virtual space. 

If I look at GDPR It’s done nothing for privacy on the web. 

An open source web will definitely face the same challenges because there is no controller on the system. My proposition is that we can control the system ourselves. 

Open source doesn't lead to enough control in order to hammer out these issues because only one person's vision is going to make this work and if there's too many visions and too many cooks in the kitchen were actually going to get a terrible user experience leading to a capitulation to a centralized virtual world owned by a cooperation and not a community.

**Comment by fire** [comment]
Copied my discussion from Discord here.

Appreciate the ask for usability.

Disclosure: These are positional statements where if I didn't want to do opensource work, I could have gotten a regular job ha. So I'm biased and invested. 

Having to read the context for points 1-13 to point 14 is hard work. So I'll work backwards from 14. Can you help me with that?

**Reply by fire** [reply]
@RedEagleP1 begins with a thesis. I'm trying to capture the essence of RedEagle's argument.

> 14 Open source may not be the answer because total freedom everywhere will just lead to chaos.

Therefore:

13 Censorship is necessary to create community.

12 Data protection will not be protected by decentralization.

11 Metaverse has the power to become a dystopian nightmare

10 Distributed governance is necessary to limit censorship

09 AR and not VR will make the Metaverse mainstream

08 Everyone will be in a single virtual world because of the network effect

07 Cannot interoperable items

06 To be usable means we cannot accommodate differences [sic]

05 Reciprocating facial expressions are necessary for organizational behavior

04 The gaming world will create the Metaverse.

03 Not being in virtual spaces creates strong relationships [sic]

02 Virtual spaces have created no significant social organization [sic]

01 The Metaverse is not possible with keyboard and mouse. [sic]

Edited:

Removed the list numbers. Restored the numbering.

**Reply by RedEagleP1** [reply]
The part of the 14 points which leads itself to the potential issues of open source are: 

4-6-7-8-12

Not to mention the issues with funding. Let me explain each in detail.

**Reply by fire** [reply]
So here are 4-6-7-8-12.

> 14 Open source may not be the answer because total freedom everywhere will just lead to chaos.

Therefore:

12 Data protection will not be aided by decentralization.

08 Everyone will be in a single virtual world because of the network effect

07 Cannot interoperable items

06 To be usable means we cannot accommodate differences [sic]

04 The gaming world will create the Metaverse.

**Reply by RedEagleP1** [reply]
Please understand that I mention these with the idea that I will be corrected and I hope to be corrected. 

4. The Metaverse will arise from a gaming world
6. Customization is the enemy of usability
7. Item interoperability is unlikely
8. The network effect would eventually push everyone to a single virtual world
12. Decentralization will not aide data protection


### # 4. The Metaverse will arise from a gaming world

Since the metaverse will arise out of a gaming world, we need a publisher of games to make the metaverse. I know that sounds convoluted but creating fun is a very difficult art and the number one comment I get from inhabitants of these metal versus is they don't know what the point of staying. One day it's like the internet and there will be a lot of points to stay there but in the meantime you need to have people cooperating and collaborating and that happens through gaming very often and gaining is a great way to break the ice between people so it's a really necessary step.

This is partially why there's so many metaverses and nothing has really taken off, they are not fun. I know I'm talking to the Second Life crowd mostly, but let's be honest, Second Life was positioned to be an incredible metaverse but it fell way below expectations and I believe the reason for that is behavioral psychology and not technology. 


### # 6. Customization is the enemy of usability

Developers tend to love lots of options, while users hate them. 

Apple is famous for taking away all the possibilities and their consumers are fanboys of products that have been falling behind for a long time (I am an Apple iOS user and I prefer it to Android). 

Gaming consoles are a great example in that it takes away the functionality of your computer that you are buying from them in order to give you much less features and charge you a price for that. There is literally no reason for gaming consoles to exist except to satisfy users that are confused by computers. This does not include Switch. 

Linux is a mine field full of distributions and is a terrible user experience on desktop. No one is developing for a platform with so many distributions because it is simply ludicrous. 

Open source games I have so many issues bringing in users because the people that develop the experience are the hardcore users and those hardcore users cater to hardcore play. I play a game called ZeroK daily because I am a hardcore gamer… it's more of a religion than a game because it's so complicated. 


### # 7. Item interoperability is unlikely

Here is another point where I think that developers vastly underestimate how confused people are by visiting virtual worlds. A virtual world has a different way of using its controls. Every virtual world has different gravity, controls, and standards (like how to know what an NPC is).

When people are divorced from the people who use their product and live in a little bubble where everyone speaks their language and their way of thinking they tend to make horrendous mistakes concerning usability. 

Recently Chrome added the copy and paste function to the menu of Chrome because so many people misunderstood how to actually copy and paste. 

Time and time again open source projects fail at usability and user acquisition due to a low budget and due to what I call the “developer bubble effect.” 

Each one of us lives in a bubble and we all experience the world in a totally unique way. We start out in life thinking that everyone is like us. However, there's a very strong distinction between people who understand things in a logical sequence and people who understand things in a very vague way. 

I'm definitely the latter, and it's very common for social people to be the latter. The reason for this is because human beings are incredibly diverse and people who are great with social skills are great at jumping between two different worlds without context. 

Most great engineers understand the world in a logical sequence of events. They have a different way of visualizing the world (they see things very differently) where they understand all the little in between bits where most of us don't and don't want to. This is why I often programmers are slow to speak compared to others. They like to know what they're saying and they're like to know how to make sure to say it very precisely. This is also why Reddit and stackoverflow are really annoying places to be for most people because everyone is so darn precise. Programmers need to be precise and being precise is too slow when it comes to jumping from person to person quickly and making connections and contacts like people like me do.

Our weekly meetings are a good example. We have tons of people and very few of them speak and that's because developers are by and large less social than your average person. 


### 8 is going to require another article 😂


### # 12. Decentralization will not aide data protection

I don't want the Metaverse to look anything like the web because the web did nothing for our privacy. The web model, in my mind, will definitely lead to oppression when put in the context of a virtual space. 

If I look at GDPR It’s done nothing for privacy on the web. 

An open source web will definitely face the same challenges because there is no controller on the system. My proposition is that we can control the system ourselves. 



Open source doesn't lead to enough control in order to hammer out these issues because only one person's vision is going to make this work and if there's too many visions and too many cooks in the kitchen were actually going to get a terrible user experience leading to a capitulation to a centralized virtual world owned by a cooperation and not a community.

**Comment by cdata** [comment]
Thanks for all of the thought-provoking comments @RedEagleP1 the conclusions one may arrive at for most of these lines of inquiry are bound to be subjective. Here are my opinions:

> 1. The Metaverse is not possible with mouse and keyboard

I don't agree that this is a given. https://hubs.mozilla.com/ is my favorite example of a multi-modal metaverse. You can use hubs from VR, or a browser tab on your desktop or phone. It is comparatively fiddly to navigate a virtual world when not in VR, but I think it is very important to be able to do this. It enables people who cannot join in an immersive capacity - as a matter of ongoing capability or momentary circumstance - to participate.

The metaverse isn't just for people who can see, or who have all their limbs, or who don't have children to take care of (and so can afford to shut out the world around them for hours at a time). It should have modalities that make it accessible to as many people as possible.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 2. No significant social organization arises out of putting people in a virtual space

This measure of the metaverse is too narrow for my liking. Minecraft is part of the metaverse. Every MMORPG is part of the metaverse. The metaverse is the reality that we are collectively hallucinating when we congregate in virtual spaces. And, there has been an incredible amount of social organization in these spaces. [People get married](https://mashable.com/article/final-fantasy-wedding), [build incredible works of art](https://westeroscraft.com/), [pick up job skills](https://money.cnn.com/2014/06/19/technology/world-of-warcraft-resume/index.html) and [execute sophisticated and verse-shattering political coups](https://www.pcgamer.com/inside-the-biggest-heist-in-eve-online-history/).

It's worth keeping in mind that just because you aren't part of the social organization doesn't mean it doesn't exist and isn't doing anything.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 3. The nature of virtual spaces leads to shallow relationships

Although many of my closest relationships were formed outside of a metaverse, most of the daily socializing I do with those people (and others) happens in a game, or on a channel like Discord. This has been especially true during the pandemic. I don't consider these relationships to be shallow just because we aren't spending time together in person every day.

I also haven't found fluid identity to be the main contributor to toxic relationships. In fact, I would go farther and say that the most toxic people I know in virtual contexts have public, "real name" identities on social media channels like Twitter and Facebook.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 4. The Metaverse will arise from a gaming world

This doesn't seem particularly controversial, although personally I consider almost any digital social context to be at least adjacent to the metaverse.

This seems orthogonal to "open source" vs. any alternative.

> 5. Reciprocating facial expressions are necessary for cooperative human behavior

I think it certainly helps, but is not necessary. There are lots of proof points for cooperative human behavior absent the ability to see each other face to face.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 6. Customization is the enemy of usability

This dynamic is a spectrum, not binary. It's definitely worth meditating on.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 7. Item interoperability is unlikely

I wrote a response to this particular section in Discord, so copying it here:

> Just to contribute some food for thought regarding NFTs:
> 
> It seems unlikely that the popular conception of NFTs will lend itself to any kind of broad-based item portability. Hot loading untrusted assets at runtime in a game presents a lot of obvious challenges and complexity (from a game developer's perspective, anyway).
>
> It does seem likely that some channels will attempt to use NFTs this way. Good luck to them. It will be interesting to see what kinds of solutions they attempt to address the challenges on the table.
>
> My theory is that we will see NFTs used in a licensing/revenue share context in a way that fosters emergent portability across games/verses. Consider this scenario:
>
> - Nike publishes a purchasable NFT for a metaverse-exclusive shoe style. The NFT includes reference art that can be displayed in a storefront, or a user profile
> - The NFT is coupled with a smart contract (perhaps one that forms a DAO) that implements a revenue sharing scheme for any channel that implements channel-native content that is faithful to the reference design
> - Game producers create art/items in the image of the reference design that fit within the technical and visual architecture of their game. When integrated, they apply for a share of the revenue from all sales of the NFT
>
> This regime would have a few virtuous qualities:
>
> - Nike can sell metaverse assets directly to customers without depending on every game/verse to mediate the sale
> - Games can be compensated for integrating support for digital assets, with the potential for income from both direct sales and resales of digital assets
> - Users can have some real expectation of item portability
>
> There are some challenges with this approach as well. For example, it becomes very difficult over time for a game producer to keep up with the sheer quantity of digital goods that could be produced. It may be possible for standardized asset formats to help with this to some extent, but it's really hard to address this in a generalized way that also matches the specific artistic look of a particular game.
>
> Another challenge is that just like licensing in the real world, there will be friction in the artistic review process, and legal headwinds when it comes to counterfeit branded content.
>
> There is always the obvious question: do you even need NFTs to make this happen?
>
> I don't necessarily think you do. But, it is hard to ignore how many brands are beginning to invest in NFTs right now, and Nike in particular just acquired an NFT avatar / clothing startup this week.
>
> The question comes down to: who controls the marketplace. There is a problem with saying: "Nike controls the marketplace." That gives Nike a ton of unilateral control/power over the marketplace. Blockchain seems like a credible neutral medium for transactions. But, we don't need blockchains to form a neutral marketplace. It is just the first technology that has presented that opportunity to us absent a legal framework.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 8. The network effect would eventually push everyone to a single virtual world

This is a matter of imagination and active investigation for many groups and individuals in this space. In order for a network effect to play out, you must have established rules for the network. Not all networks centralize; many settle into a meta-stable state until disruptive forces change the status quo. In order to know for certain that a network will centralize, you must already have an architecture in mind for the network.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 9. AR and not VR will make the Metaverse mainstream

This isn't necessarily controversial to me, but it is very subjective.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 10. Distributed governance is necessary to limit censorship

I think it helps, but also distributed governance can still result in tyranny depending on the details of the architecture.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 11. The Metaverse has the power to become a dystopian nightmare

Yes, but I would add that this applies to networked communications technology in general.

Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

> 12. Decentralization will not aide data protection

This needs more specifics in order for me to form an opinion: decentralization across what axis?

This seems orthogonal to "open source" vs. any alternative.

> 13. Censorship is necessary to create community

I don't necessarily disagree, but this seems orthogonal to "open source" vs. any alternative

> 14. Open source may not be the answer

I guess what I'm missing here is: what answer are you looking for?

You might be right, but points 1-13 seem like they would apply whether open source is part of the equation or not.

**Comment by RedEagleP1** [comment]
> I don't agree that this is a given. https://hubs.mozilla.com/ is my favorite example of a multi-modal metaverse. You can use hubs from VR, or a browser tab on your desktop or phone. It is comparatively fiddly to navigate a virtual world when not in VR, but I think it is very important to be able to do this. It enables people who cannot join in an immersive capacity - as a matter of ongoing capability or momentary circumstance - to participate.
> 
> The metaverse isn't just for people who can see, or who have all their limbs, or who don't have children to take care of (and so can afford to shut out the world around them for hours at a time). It should have modalities that make it accessible to as many people as possible.
> 
> Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative.

Orthogonal?

I believe you are conflating two very different things here. You are saying that the metaverse requires mouse and keyboard to be ethical and accessible and what I'm saying is that for most people the usability of the metaverse is way too low when it comes to mouse and keyboard. This is partially due to the developer bubble effect I mentioned earlier: 

> 
> Time and time again open source projects fail at usability and user acquisition due to a low budget and due to what I call the “developer bubble effect.” 
> 
> Each one of us lives in a bubble and we all experience the world in a totally unique way. We start out in life thinking that everyone is like us. However, there's a very strong distinction between people who understand things in a logical sequence and people who understand things in a very vague way. 
> 
> I'm definitely the latter, and it's very common for social people to be the latter. The reason for this is because human beings are incredibly diverse and people who are great with social skills are great at jumping between two different worlds without context. 
> 

Because engineer's understand the world in a logical sequence of events it's very normal for them to look up the keybinds of a game before they play. No normal human being does this and that's because they hate having to understand everything from the ground up. 

Please understand I say that in no derogatory form whatsoever. Programmers have super powers I could ever have. However I have also noticed that they make up a very small percentage of the general population. I think with the amount of money programmers make these days we would all be programmers if we have the option. 

>This measure of the metaverse is too narrow for my liking. 

The definition is far from settled I would say. In this particular article I am referencing only the future of human existence and civilization. 

> I think it certainly helps, but is not necessary. 

I actually agree with this, my hope was to stimulate a little discussion by taking a very strong stance rather than a mediocre one. 😄 

> Regardless of whether or not you agree: this seems orthogonal to "open source" vs. any alternative. 

You should read what I replied to iFire, it might help. 

Re: # 7, your code block is not really readable due to having to scroll right and left. 

> I think it helps, but also distributed governance can still result in tyranny depending on the details of the architecture.

I agree.

**Comment by cdata** [comment]
"Open source" is just a means to an end. There are other models to achieve the end. None of the models seem immune from the problems you are describing. This is why I say your problem statement is orthogonal to "open source."

Everyone lives in a bubble defined by their subjective lens on reality. And, no-one can be defined comprehensively by narrow archetypes like, "developer," "project manager," "chef," or "parent." We each contain multitudes.

Thanks for the thought-provoking suggestions. I think I'll let my first answers stand.

**Reply by RedEagleP1** [reply]
What I am doing now is exploring other models but I am told that there are open source means of reaching the goal of freedom in the Metaverse. I just don't quite know how to get there.  

And thank you as well!
